api/v1beta1/module_webhook_test.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"testing"
+)
+
+func TestV1beta1(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "V1beta1 Suite")
+}
+
+var validKMs = []KernelMapping{
+	{Regexp: "valid-regex"},
+}
+
+var invalidKMs = []KernelMapping{
+	{Regexp: "*-invalid-regex"},
+}
+
+var module = &Module{
+	Spec: ModuleSpec{
+		ModuleLoader: ModuleLoaderSpec{
+			Container: ModuleLoaderContainerSpec{
+				KernelMappings: nil,
+			},
+		},
+	},
+}
+
+var _ = BeforeEach(func() {
+	module.Spec.ModuleLoader.Container.KernelMappings = validKMs
+})
```

Comment:
I will suggest not having a global `BeforeEach`.

You can have 1 BeforeEach per spec (`It`) if you wish if it makes sense to have it. Otherwise I will just defined a module in each `Describe` or `It` separately.